cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -74,7 +109,11 @@ message CassandraNode {
     optional CassandraNodeTask metadataTask = 6;
     optional CassandraNodeTask serverTask = 7;
 
-    repeated CassandraNodeHealthCheckDetails healthCheckHistory = 8;
+    repeated HealthCheckDetails healthCheckHistory = 8;
+
+    required bool seed = 9;
```

Comment:
I'd prefer to not flag nodes as seeds but instead have a seeds list in the CassandraCluster object. I'm under the impression that seeds won't change often.